### DISCUSSION TITLE
_a2.find is not a function

### DISCUSSION THREAD

**Discussion by PyrkinValentin** [heroui-inc/heroui#3760] [Help]
Hello. I use nextui and I really like it. Today I updated all nextui components to the new version and now there is a problem.

The specific problem is in the Dropdown component.
Dropdown version: ^2.1.29 -> ^2.1.30

Line 101: const item = (_a2 = props2 == null ? void 0 : props2.children) == null ? void 0 : _a2.find((item2) => item2.key === key);
Could you help solve this problem (_a2.find is not a function)?

Best regards!

**Comment by wingkwong** [comment]
able to reproduce. converting to an issue.

**Comment by wingkwong** [comment]
fixed in https://github.com/nextui-org/nextui/pull/3762
